src/components/widgets-settings/accounts-list/accounts-list.directive.coffee

```diff
@@ -33,9 +33,8 @@ module.controller('SettingAccountsListCtrl', ($scope, $timeout, ImpacWidgetsSvc)
 
     if w.content? && !_.isEmpty(w.content.complete_list)
       w.remainingAccounts = angular.copy(w.content.complete_list)
-      $timeout () ->
```

Comment:
@an-pavlovych the timeout was there to ensure all the context is passed to the directive before executing the next statements.
Any particular reason for removing it?